### PR TITLE
Phase2-hcx221 define Trigger DetId class for HFNose

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Define Trigger DetId class for HFNose. Required to study L1 trigger using HFNose

#### PR validation:

Tested using standard matrix

#### if this PR is a backport please specify the original PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by civanch** [comment]
+1

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: DataFormats/ForwardDetId/interface/ForwardSubdetector.h [hunk 1]
```diff
@@ -11,6 +11,6 @@ enum ForwardSubdetector {
   HFNose = 6,
   HGCTrigger = 7
 };
-enum HGCalTriggerSubdetector { HGCalEmptyTrigger = 0, HGCalEETrigger = 1, HGCalHSiTrigger = 2, HGCalHScTrigger = 3 };
+enum HGCalTriggerSubdetector { HFNoseTrigger = 0, HGCalEETrigger = 1, HGCalHSiTrigger = 2, HGCalHScTrigger = 3 };
```
[the remark this answers is not in the document]
Comment (jbsauvan) [reply DataFormats/ForwardDetId/interface/ForwardSubdetector.h]: Hi @kpedro88 
No, it is not used and there is no plan to use it.

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by kpedro88** [comment]
+upgrade

**Comment by bsunanda** [comment]
@civanch Please approve this

**Comment by civanch** [comment]
+1

**Comment by kpedro88** [comment]
+1

**Pull request merged**